api_spec.yaml

```diff
@@ -3246,9 +3246,496 @@ paths:
           $ref: '#/components/responses/401'
         '500':
           $ref: '#/components/responses/500'
-          
-      
+  
+  /recipe/multitenancy/tenant:
+    put:
+      tags:
+        - Multitenancy Recipe
+      description: |
+        Creates or updates a tenant.
+
+        If `tenantId` is not specified, `defaultTenantId` will be used.
```

Comment:
change defaulttenantid to public everywhere

Reply:
Done